app/models/area_definition.rb

```diff
@@ -1,4 +1,10 @@
 class AreaDefinition < ApplicationRecord
-  belongs_to :area
   belongs_to :zip_code
+
+  # Need to set 'optional: true' because otherwise Rails assumes the record
```

Comment:
Is this only for running the rake task? I wonder if there's a way to bypass that during the import and not mark these as optional?

Reply:
Yes, only for this scenario, presuming area definitions are only defined by us on an as-needed basis (like through that rake task). Didn't think to skip validations, but I'll try that.